homeassistant/const.py

```diff
@@ -767,8 +767,23 @@ class UnitOfMass(StrEnum):
 # Conductivity units
 CONDUCTIVITY: Final = "µS/cm"
 
+
 # Light units
+class UnitOfLuminousFlux(StrEnum):
+    """Luminous flux units."""
+
+    LUMEN = "lm"
```

Comment:
This is a new constant, to replace local constants in `plugwise`, `tuya`, `xiaomi_aqara` and `xiaomi_miio`

Reply:
There is also this documentation:
https://en.wikipedia.org/wiki/Candela_per_square_metre#See_also